src/components/Food-item-card/FoodItem.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"; //17.0.2
+import  { useState } from "react";
+import PropTypes from "prop-types";
+import "./style.css";
+import {BrowserRouter as Router,Link} from "react-router-dom";
+function FoodItem(FoodItem) {
```

Comment:
please replace the props name with something more reasonable not the components name
```suggestion
function FoodItem(props) {
```